src/main/java/com/synopsys/integration/alert/web/config/JobConfigActions.java

```diff
@@ -61,6 +61,7 @@
 
 @Component
 public class JobConfigActions {
+    public static final String CUSTOM_MESSAGE = "-CUSTOM_MESSAGE";
```

Comment:
Why the dash? The name of this variable really doesn't give me any insight into how it's used. Maybe go with `CUSTOM_MESSAGE_SUFFIX`.